src/applications/ivc-champva/10-10D/helpers/utilities.js

```diff
@@ -44,7 +44,7 @@ export const applicantListSchema = (requireds, propertyList) => {
 // Extracting this to a function so there aren't a thousand identical
 // ternaries we have to change later
 export function sponsorWording(formData, isPosessive = true, cap = true) {
-  const retVal = isPosessive ? 'sponsor’s' : 'sponsor';
+  const retVal = isPosessive ? 'veteran’s' : 'veteran';
```

Comment:
Veteran should be capitalized. It is a proper noun according to [DST guidance](https://design.va.gov/content-style-guide/capitalization).